BurnOutSharp/ProtectionType/Tages.cs

```diff
@@ -20,110 +20,190 @@ public string CheckContents(string file, byte[] fileContent, bool includeDebug,
+                    ./Wave.apt (0351d0f3d4166362a1a9d838c9390a3d92945a44)
+                    ./Wave.axt (SHA-1 is variable, file size is 61,440 bytes)
+                    */
+                    Path.Combine("Sys", "Devx.sys").Replace("\\", "/"),
+                    Path.Combine("Sys", "VtPr.sys").Replace("\\", "/"),
+                    Path.Combine("Wave.aif").Replace("\\", "/"),
+                    Path.Combine("Wave.alf").Replace("\\", "/"),
+                    Path.Combine("Wave.apt").Replace("\\", "/"),
+                    Path.Combine("Wave.axt").Replace("\\", "/"),
+                }, "TAGES"),
+
+                new PathMatchSet(new List<string>
+                {
+                    /*
+                    Currently only known to exist in "XIII" and (presumably) "Beyond Good & Evil".
+                    */
+                    Path.Combine("enodpl.sys").Replace("\\", "/"),
+                    Path.Combine("ENODPL.VXD").Replace("\\", "/"),
+                    Path.Combine("tandpl.sys").Replace("\\", "/"),
+                    Path.Combine("TANDPL.VXD").Replace("\\", "/"),
+                }, "TAGES"),
+
+                new PathMatchSet(new List<string>
+                {
+                    /*
+                    The directory of these files has been seen to be named two different things, with two different accompanying executables in the root of the directory.
+                    In the example where the directory is named "Drivers", the executable is named "Silent.exe".
+                    In the example where the directory is named "ELBdrivers", the executable is name "ELBDrivers.exe".
+
+                    Expected information about these checks:
+                    ./9x/hwpsgt.vxd (40826e95f3ad8031b6debe15aca052c701288e04)
+                    ./9x/lemsgt.vxd (f82339d797be6da92f5d9dadeae9025385159057)
+                    ./NT/hwpsgt.sys (43f407ecdc0d87a3713126b757ccaad07ade285f)
+                    ./NT/lemsgt.sys (548dd6359abbcc8c84ce346d078664eeedc716f7)
+                    (The name and file size of the included executable vary, but there should always be one here.)
+                    */
+                    Path.Combine("9x", "hwpsgt.vxd").Replace("\\", "/"),
+                    Path.Combine("9x", "lemsgt.vxd").Replace("\\", "/"),
+                    Path.Combine("NT", "hwpsgt.sys").Replace("\\", "/"),
+                    Path.Combine("NT", "lemsgt.sys").Replace("\\", "/"),
+                }, "TAGES"),
+            };
+
+            return MatchUtil.GetAllMatches(files, matchers, any: true);
+        }
 
-            // "Wave.aif" was a former check here, but it over-matched
-            if (files.Any(f => Path.GetFileName(f).Equals("Tages.dll", StringComparison.OrdinalIgnoreCase)))
+        /// <inheritdoc/>
+        public string CheckFilePath(string path)
+        {
+            if (Path.GetFileName(path).Equals("Devx.sys", StringComparison.OrdinalIgnoreCase))
```

Comment:
Please try to use the new path check framework here instead of the old `if/else if` format.